### DISCUSSION TITLE
Unified Search

### DISCUSSION THREAD

**Discussion by jflattery** [advplyr/audiobookshelf#2422] [Ideas]
IMO there should be an ability to have a unified search box, such that if I search for `Michio Kaku`, I should see a list of his books as well as podcast episodes that he was either on or at least mentioned in the episode `Title`, `Subtitle`, or `Description`.

**Comment by advplyr** [comment]
There is a FR https://github.com/advplyr/audiobookshelf/issues/2236

**Reply by jflattery** [reply]
Although, I'm not sure if I would like it visually to appear as u unified library. I like having podcasts separate from audiobooks. It would be just nice to search for a person or topic and see related content.

**Reply by advplyr** [reply]
Also got that one here https://github.com/advplyr/audiobookshelf/issues/556
